app/models/trainee.rb

```diff
@@ -3,6 +3,7 @@
 class Trainee < ApplicationRecord
   include Sluggable
   include PgSearch::Model
+  include Discard::Model
```

Comment:
I think we need to scope a few places with the `Trainee.kept`, eg, in the relationships, filters, and controllers.
As of right now, I'm able filter by name/view discarded trainees, which might not be desired.
(here is a trainee that I tried with)
https://register-pr-1718.london.cloudapps.digital/trainees/rdwPbTtnCVf84SBmzKZ6hqSu

Reply:
Good spot.